lib/scss_lint/linter/README.md

```diff
@@ -810,7 +810,7 @@ execute `:sort` in `vim`), and it can
 You can also specify an explicit ordering via the `order` option, which allows
 you to specify an explicit array of properties representing the preferred
 order, or the name of a
-[preset order](data/property-sort-orders).
+[preset order](../../../../master/data/property-sort-orders).
```

Comment:
it sorta does, but it's sorta ugly this way, too. I almost prefer the hardcoded link, it's at least readable at a glance.